spec.html

```diff
@@ -8854,13 +8854,15 @@ <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _protot
     </emu-clause>
 
     <emu-clause id="sec-createbuiltingeneratorinstance" aoid="CreateBuiltinGeneratorInstance">
-      <h1>CreateBuiltinGeneratorInstance ( _closure_ [ , _generatorBrand_ [ , _generatorPrototype_ ] ] )</h1>
-      <p>The abstract operation CreateBuiltinGeneratorInstance takes arguments _closure_ (an Abstract Closure with no parameter), optional argument _generatorBrand_, and _generatorPrototype_ (an Object).</p>
+      <h1>CreateBuiltinGeneratorInstance ( _closure_ [ , _generatorBrand_ [ , _generatorPrototype_ [, _generatorSlots_] ] ] )</h1>
```

Comment:
The lint is failing because of a missing ` `.
```suggestion
      <h1>CreateBuiltinGeneratorInstance ( _closure_ [ , _generatorBrand_ [ , _generatorPrototype_ [ , _generatorSlots_] ] ] )</h1>
```